spotbugs/src/main/java/edu/umd/cs/findbugs/classfile/engine/SelfMethodCalls.java

```diff
@@ -41,7 +41,7 @@
 public class SelfMethodCalls {
 
     static private boolean interestingSignature(String signature) {
-        return !"()V".equals(signature);
+        return true;
     }
```

Comment:
As far as I can see, this function is only used once, in line 65. Since it will always return true, IMO it's better to remove this part of the condition.